apps/ai-image-generator/backend/src/utils.ts

```diff
@@ -19,17 +24,31 @@ export const toSharp = (imageStream: NodeJS.ReadableStream): sharp.Sharp => {
   return imageStream.pipe(sharpStream);
 };
 
+const difference = (a: number, b: number): number => {
```

Comment:
Personal nit: I think this abstraction is unnecessary.

Reply:
Honestly it was doing my head in a bit thinking about which two values i was comparing and this helped me reason about the parent function a lot better.